src/main/java/baseball/BaseBallGame.java

```diff
@@ -0,0 +1,98 @@
+package baseball;
+
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class BaseBallGame {
+    private ArrayList<Integer> answerNum = new ArrayList<>();
+    private ArrayList<Integer> userNum = new ArrayList<>();
+    private GameState state = GameState.NOT_START;
+
+    public void startGame() {
+        System.out.println("숫자 야구 게임을 시작합니다.");
+        state = GameState.PLAYING;
+
+        play();
+    }
+
+    private void play() {
+        generateNum();
+        while (state != GameState.CORRECT) {
+            chooseNum();
+            System.out.println(getResult());
+        }
+        exitGame();
+        if (state == GameState.PLAYING) {
+            play();
+        }
+    }
+
+
+    private void chooseNum() {
+        String input = "";
+
+        System.out.print("숫자를 입력해주세요 : ");
+        input = Console.readLine();
```

Comment:
```suggestion
        System.out.print("숫자를 입력해주세요 : ");
        String input = Console.readLine();
```
이런 식으로 수정하면 로직 라인을 줄일 수 있을 것 같습니다!